File: Entrega-2/src/laboratorio/StrategyJuniorRobot.java
package laboratorio;



public interface StrategyJuniorRobot {

	void run(Blitzcrank aJuniorRobot);

	void onHitWall(Blitzcrank aJuniorRobot);

	void onHitByBullet(Blitzcrank aJuniorRobot);

	void onScannedRobot(Blitzcrank aJuniorRobot);

}

File: Entrega-1-ko/src/laboratorio/StrategyJuniorRobot.kt
package laboratorio

import robocode.JuniorRobot

interface StrategyJuniorRobot {
    fun run(aJuniorRobot: JuniorRobot)

    fun onHitWall(aJuniorRobot: JuniorRobot)

    fun onHitByBullet(aJuniorRobot: JuniorRobot)

    fun onScannedRobot(aJuniorRobot: JuniorRobot)
}
